Scripts/Editor/XRLineRendererEditor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using UnityEngine;
+
+namespace UnityEditor
+{
+    [CustomEditor(typeof(XRLineRenderer))]
+    [CanEditMultipleObjects]
+    public class XRLineRendererEditor : Editor
+    {
+        SerializedProperty m_Materials;
+        SerializedProperty m_Positions;
+        SerializedProperty m_UseWorldSpace;
+        SerializedProperty m_Loop;
+        SerializedProperty m_Width;
+        SerializedProperty m_WidthCurve;
+        SerializedProperty m_Color;
+
+        void OnEnable()
+        {
+            m_Materials = serializedObject.FindProperty("m_Materials");
+            m_Positions = serializedObject.FindProperty("m_Positions");
+            m_UseWorldSpace = serializedObject.FindProperty("m_UseWorldSpace");
+            m_Loop = serializedObject.FindProperty("m_Loop");
+            m_Width = serializedObject.FindProperty("m_Width");
+            m_WidthCurve = serializedObject.FindProperty("m_WidthCurve");
+            m_Color = serializedObject.FindProperty("m_Color");
+        }
+
+        public override void OnInspectorGUI()
+        {
+            serializedObject.Update();
+            EditorGUILayout.PropertyField(m_Materials, true);
+            EditorGUILayout.PropertyField(m_Positions, true);
+
+            using (new EditorGUI.DisabledGroupScope(true))
```

Comment:
Why is this always disabled? Its tooltip says it's driven by the shader but it still looks like it can just be set from code. Why not be able to set it from the Inspector? If there's a good reason not to allow setting it from the Inspector, then I don't think you should show it in the first place.